install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -570,6 +570,9 @@ data:
 {{- if hasKey .Values.loadBalancer "dsrDispatch" }}
   bpf-lb-dsr-dispatch: {{ .Values.loadBalancer.dsrDispatch | quote }}
 {{- end }}
+{{- if hasKey .Values.loadBalancer "serviceTopology" }}
+  enable-service-topology: {{ .Values.loadBalancer.serviceTopology | quote }}
```

Comment:
We should document this properly by adding it (commented out) to `values.yaml` too (check the other `loadBalancer` fields for an example)